ext/devcontainer/devcontainer-feature.json

```diff
@@ -0,0 +1,26 @@
+{
+    "id": "azure-dev-cli",
+    "version": "1.0.0",
+    "name": "Azure Dev CLI",
+    "documentationURL": "https://github.com/azure/azure-dev/tree/main/ext/devcontainer",
+    "description": "Installs the Azure Developer CLI along with needed dependencies.",
+    "options": {
+      "version": {
+        "type": "string",
+        "proposals": [
+          "latest"
+        ],
+        "default": "latest",
+        "description": "Select or enter an Azure Developer CLI version. (Available versions may vary by Linux distribution.)"
+      }
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+              "ms-azuretools.azure-dev",
```

Comment:
Sorry, I should have mentioned this earlier: Azure Pack contains the Azure dev CLI extension already: https://github.com/microsoft/vscode-node-azure-pack/blob/main/package.json so this line is redundant.

Reply:
Well, should we include Azure Pack then? A feature should be fairly small and contained for what it offers and Azure Pack really brings in a lot of stuff that's generally related to Azure, but not specific for the feature.

Reply:
We are working on making the Azure Pack and AZD more integrated and the experience more polished and approachable; it should land pretty soon. I really think the Pack will give people the best starting point.